### ISSUE TITLE
vue-a11y causes a conflict with vue/require-component-is rule

### ISSUE THREAD

**Issue by wujekbogdan** [maranran/eslint-plugin-vue-a11y#29]
**Describe the bug**

When I use `eslint-plugin-vue-a11y` together with `eslint-plugin-vue` then `vue/require-component-is` rule fails to validate the `:is` property. Even if the `:is` property exists I'm getting ESLint error.

**To Reproduce**

1. Add `plugin:vue/recommended` and `plugin:vue-a11y/recommended` to ESLint config's `extends` section
2. Use a `component` tag together with `:is`
3. ESLint errors shows up.

**Expected behavior**
There should be no error if the `:is` property has a proper value.

**Screenshots**

When `plugin:vue-a11y/recommended` is active:

<img width="1121" alt="Zrzut ekranu 2020-04-21 o 14 52 01" src="https://user-images.githubusercontent.com/533954/79874211-f9cb0900-83df-11ea-808a-a868499134b6.png">

Without `plugin:vue-a11y/recommended`

<img width="1109" alt="Zrzut ekranu 2020-04-21 o 14 54 51" src="https://user-images.githubusercontent.com/533954/79874382-2d0d9800-83e0-11ea-9138-c468ff44c5dc.png">

**Desktop (please complete the following information):**
 - OS: macOS
 - Version `eslint-plugin-vue^6.2.2`, `eslint-plugin-vue-a11y^0.0.31`
